packages/backend/src/services/UserService.ts

```diff
@@ -848,4 +849,46 @@ export class UserService {
             },
         });
     }
+
+    private static async generateGoogleAccessToken(
+        refreshToken: string,
+    ): Promise<string> {
+        return new Promise((resolve, reject) => {
+            refresh.requestNewAccessToken(
+                'google',
+                refreshToken,
+                (err: any, accessToken: string) => {
+                    if (err || !accessToken) {
+                        reject(err);
+                        return;
+                    }
+                    resolve(accessToken);
+                },
+            );
+        });
+    }
+
+    /**
+     * This method is used on the gdrive API to get the accessToken for listing files on the user's drive
+     * @param user
+     * @returns accessToken
+     */
+    async getAccessToken(user: SessionUser): Promise<string> {
```

Comment:
I wanted to do this on `authentication` file, but we call useService in there, so this was going to cause circular dependencies

Reply:
`Pick<SessionUser, 'userUuid'>` then you can reuse?